app/test_main.py

```diff
@@ -2,2 +1,23 @@
+from app.main import get_human_age, convert_to_human
 
-# write your code here
+
+def test_convert_to_human_age() -> None:
+    assert convert_to_human(0, 15, 9, 4) == 0
+    assert convert_to_human(14, 15, 9, 4) == 0
+    assert convert_to_human(15, 15, 9, 4) == 1
+    assert convert_to_human(23, 15, 9, 4) == 1
+    assert convert_to_human(24, 15, 9, 4) == 2
+    assert convert_to_human(27, 15, 9, 4) == 2
+    assert convert_to_human(28, 15, 9, 4) == 3
+    assert convert_to_human(100, 15, 9, 4) == 21
+
+
+def test_get_human_age() -> None:
+    assert get_human_age(0, 0) == [0, 0]
+    assert get_human_age(14, 14) == [0, 0]
```

Comment:
The test cases for `get_human_age` with inputs `(0, 0)` and `(14, 14)` both expect the same output `[0, 0]`. Consider removing one of these to avoid repetition as per the checklist requirement: "Make sure that your tests don't test the same case few times."